addons/mail/static/src/core/common/link_preview.xml

```diff
@@ -52,43 +52,52 @@
-                            <div class="position-absolute top-50 start-50 translate-middle">
-                                <div class="o-mail-LinkPreviewVideo-play btn btn-lg rounded opacity-75 opacity-100-hover transition-base">
-                                    <i class="fa fa-play"/>
-                                </div>
-                            </div>
-                        </div>
-                        <span t-else="" class="d-flex align-items-center justify-content-center w-100 h-100 bg-100 text-300">
-                            <i class="fa fa-camera fa-2x"/>
-                        </span>
-                    </a>
+                <div t-if="!props.linkPreview.videoURL" class="o-mail-LinkPreviewVideo-videoWrap col-4 d-block">
+                    <a t-att-href="props.linkPreview.source_url" target="_blank"><t t-call="mail.LinkPreview.videoPreview"/></a>
                 </div>
             </div>
+            <div t-if="props.linkPreview.videoURL" class="o-linkPreviewVideo-container">
+                <div t-if="!state.videoLoaded" class="o-mail-LinkPreviewVideo-videoWrap h-100 w-100" t-on-click="() => state.startVideo = true"><t t-call="mail.LinkPreview.videoPreview"/></div>
+                <iframe t-if="state.startVideo" t-att-src="props.linkPreview.videoURL" class="h-100 w-100" t-att-class="{'d-none': !state.videoLoaded}" allow="autoplay" frameborder="0" allowFullScreen="true" t-ref="video"/>
+            </div>
             <t t-if="props.deletable" t-call="mail.LinkPreview.aside">
                 <t t-set="className" t-value="'fa fa-stack p-0 opacity-75 opacity-100-hover'"/>
             </t>
         </div>
     </t>
-    
+
     <t t-name="mail.LinkPreview.aside">
         <div class="position-absolute top-0 end-0 small">
             <button t-attf-class="{{ className }}" class="o-mail-LinkPreview-aside btn" aria-label="Remove" t-on-click="onClick">
                 <i class="fa fa-times"/>
             </button>
         </div>
     </t>
+
+    <t t-name="mail.LinkPreview.videoPreview">
+        <div  class="o-mail-LinkPreviewVideo-overlay position-relative h-100" t-att-class="{'bg-dark': !props.linkPreview.og_image}">
+            <img t-if="props.linkPreview.og_image" class="img-fluid h-100 rounded-bottom o_object_fit_cover" t-att-src="props.linkPreview.og_image" t-att-alt="props.linkPreview.og_title" t-on-load="onImageLoaded"/>
+            <i t-elif="props.linkPreview.videoURL" class="fa fa-television fa-8x position-absolute top-50 start-50 translate-middle"/>
+            <span t-else="" class="d-flex align-items-center justify-content-center w-100 h-100 bg-100 text-300">
+                <i class="fa fa-camera fa-2x"/>
+            </span>
+            <div class="position-absolute top-50 start-50 translate-middle">
+                <div t-if="!state.startVideo" class="o-mail-LinkPreviewVideo-play btn btn-lg rounded opacity-75 opacity-100-hover transition-base">
```

Comment:
I would put `bg-white rounded` or equivalent on `div.position-absolute.top-50.start-50.translate-middle`.
This barely change style in dark theme which is good. In white theme this makes the `.fa-play` more visible, which gives a better clue this is a video player. Otherwise this is too subtle and I image many people will miss it.
Here's a comparison before/after, which a youtube embed video to give a fair comparison:

<img width="525" alt="Screenshot 2024-10-01 at 01 57 28" src="https://github.com/user-attachments/assets/ea442d73-8574-4d0f-a8d6-7f3241367917">